src/components/app.tsx

```diff
@@ -279,7 +279,7 @@ export class App extends React.PureComponent<IProps, IState> {
                                : activityIndex !== undefined && activityIndex >= 0
                                  ? activityIndex + 1
                                  : undefined;
-    sequenceActivity !== undefined && setQueryValue("sequenceActivity", sequenceActivity);
+    sequenceActivity !== undefined && setQueryValue("sequenceActivity", "activity_" + sequenceActivity);
```

Comment:
I'm still not sure this matches the readme:

```
* sequenceActivity={n|activity_[id]}: load activity n where n corresponds to the activity's placement in the order of sequenced activities (1 = first activity, 2 = second activity, etc.), or by the activity's unique ID
* page={n|"page_[id]"}:               load page n, where 0 is the activity introduction, 1 is the first page and 
```

If I understand the code above, the `sequenceActivity` variable will be the placement of the the activity in the sequence. So that means it shouldn't be prefixed with `activity_` when added as a URL parameter.

Reply:
We can either set it to placement or index. Changed the logic on line 278 to be more inline with the readme.

Reply:
Fixed this bug. The conversion script wasn't copying over the activity ids, so the sequence couldn't find an id and was returning activity position instead.

Reply:
looks good.